Changelog.md

```diff
@@ -1,6 +1,7 @@
 # FOSSA CLI Changelog
 
-## Unreleased
+## 3.9.46
+- Licensing: Fix a bug where the tzdata debian package copyright was not detected as a public domain license
```

Comment:
Can you add a link to the PRs? E.g. for the first one:

```
- Licensing: Fix a bug where the tzdata debian package copyright was not detected as a public domain license ([#1504](https://github.com/fossas/fossa-cli/pull/1504))
```

Reply:
Oops; done!